lib/model/autocomplete.dart

```diff
@@ -284,46 +284,33 @@ abstract class AutocompleteView<QueryT extends AutocompleteQuery, ResultT extend
   }
 }
 
-class MentionAutocompleteView extends AutocompleteView<MentionAutocompleteQuery, MentionAutocompleteResult, User> {
+class MentionAutocompleteView extends AutocompleteView<MentionAutocompleteQuery, MentionAutocompleteResult> {
   MentionAutocompleteView._({
     required super.store,
     required this.narrow,
+    required this.wildcards,
     required this.sortedUsers,
   });
 
   factory MentionAutocompleteView.init({
     required PerAccountStore store,
     required Narrow narrow,
+    required List<Wildcard> wildcards,
   }) {
     final view = MentionAutocompleteView._(
       store: store,
       narrow: narrow,
+      wildcards: wildcards,
       sortedUsers: _usersByRelevance(store: store, narrow: narrow),
     );
     store.autocompleteViewManager.registerMentionAutocomplete(view);
     return view;
   }
 
   final Narrow narrow;
+  final List<Wildcard> wildcards;
   final List<User> sortedUsers;
 
-  @override
-  Future<List<MentionAutocompleteResult>?> computeResults() async {
```

Comment:
Moving this method to lower in the file is perfectly reasonable, but should happen as its own separate NFC prep commit. That way the substantive change stands out better.